### PR TITLE
Bug fix 3.4/foxxmaster test

### PR DESCRIPTION
**Pull request by fceller**

### CODE REVIEW

**Comment by fceller** [comment]
@graetzer it was indeed the wrong branch, fixed

File: tests/js/client/resilience/foxxmaster.js [hunk 1]
```diff
@@ -96,7 +96,6 @@ function FoxxmasterSuite() {
   return {
     setUp: function() {
       serverSetup();
-      wait(2.1);
```
Comment (fceller) [inline tests/js/client/resilience/foxxmaster.js]: I'm unsure if this is needed or not

Comment (fceller) [reply tests/js/client/resilience/foxxmaster.js]: @graetzer @mchacki

Comment (graetzer) [reply tests/js/client/resilience/foxxmaster.js]: hm seems like a very specific timeout. Maybe to allow for self-healt to install the foxx app everywehere

**Comment by KVS85** [comment]
Overtaken by #6654.

**Pull request closed without merging**